src/saga/adaptors/pbspro/pbsprojob.py

```diff
@@ -312,6 +312,12 @@ def _pbscript_generator(url, logger, jd, ppn, gres, pbs_version, is_cray=False,
     elif 'PBSPro_13' in pbs_version:
         logger.info("Using PBSPro 13 notation '#PBS -l select=XX' ")
         pbs_params += "#PBS -l select=%d\n" % (nnodes)
+    elif '14.2' in pbs_version:
```

Comment:
Is cheyenne using 14.2?  If so, you probably want to check for the hostname first, and then for the pbs version?

Reply:
Why 'and' specifically? What if there is another machine that uses 14.2, the condition will fail.

Reply:
If cheyenne is using 14.2, then the cheyenne elif branch in this if-tree will never be reached, because this one applies first.

Reply:
Ah you're right. Fixed now. 

Question: Do you know why SAGA_PPN gets the value of 72 in the case of cheyenne? ``ppn`` is actually 36 on Cheyenne. I am asking since (1) that is the only difference between the two elif conditions and (2) I am not yet sure if that will break anything in RP.